spec/controllers/application_templates_controller_spec.rb

```diff
@@ -109,4 +109,47 @@
       end
     end
   end
+
+  describe 'POST #set_active' do
+    before :each do
+      @template = create :application_template
+    end
+    let :submit do
+      post :set_active, id: @template.id
+    end
+    context 'student' do
+      it 'does not allow access' do
+        when_current_user_is :student
+        submit
+        expect(response).to have_http_status :unauthorized
+      end
+    end
+    context 'staff' do
+      before :each do
+        when_current_user_is :staff
+      end
+      context 'button to reactivate application template has been pressed' do
+        let :submit do
+          post :set_active, id: @template.id, up: 'value'
+        end
+        it 'marks the application as active' do
+          submit
+          expect(@template.reload.active).to eql true
```

Comment:
```ruby
expect(@template.reload).to be_active
```

should work too (you could even try it without the `reload` to see if that works).